radix-transactions/src/manifest/compiler.rs

```diff
@@ -25,21 +25,44 @@ pub fn compile_error_diagnostics(
     }
 }
 
-pub fn compile<B>(
+pub use compile_manifest_v1 as compile;
```

Comment:
Is this `compile` used anywhere?
If yes then how about renaming to `compile_manifest` for easier grepping (current `compile_manifest` could be  merged with `compile_manifest_v1`)?

Reply:
It could be used by downstream clients. But perhaps it's okay to remove. Will remove...